### PR TITLE
Remove text knob from input-provider

### LINKED ISSUE #1598: Input Provider with Storybook knobs not working
**Describe the bug**
The input provider drop-down is broken on `latest`
Screen Shot input provider broken

**To Reproduce**
Steps to reproduce the behaviour:

1. Go to 
https://bbc.github.io/psammead/?path=/story/components-headline--default
https://bbc.github.io/psammead/?path=/story/components-inlinelink--default
2. On the 'knobs' panel, select a service that is not 'news'. e.g `arabic`.
3. See just right-aligned english copy.

This is the case for all stories that use knobs and the inputProvider in psammead-storybook-helpers

**Expected behaviour**
When you select a service that is not `news`, you should get copy written in the relevant language. 

**Screenshots**
Pasted above. 

**Additional context**
Way to see it working again: Pin to version `5.0.6`. 
However, it should be fixed forwards instead of pinned to this older version.

### PR DESCRIPTION
**Pull request by ghost**
Resolves #1598

**Overall change:** Remove text knob from `input-provider`

**Code changes:**

- Remove `text` knob from `input-provider`

See https://github.com/bbc/psammead/issues/1598#issuecomment-519141487 and https://github.com/bbc/psammead/issues/1598#issuecomment-519182428. Suggestions on alternative ways to solve this are highly welcomed.

---

- [x] I have assigned myself to this PR and the corresponding issues
- [ ] Tests added for new features
- [ ] Test engineer approval

### CODE REVIEW

**Review by DenisHdz — APPROVED** [review]
LGTM, as long as everyone is happy about losing this functionality.

File: packages/utilities/psammead-storybook-helpers/src/input-provider.jsx [hunk 1]
```diff
@@ -16,18 +16,15 @@ const inputProvider = (slots, componentFunction, services) => () => {
...
-      // text for the knob to the snippet from LANGUAGE_VARIANTS for that
-      // language.
-      defaultText && isNews ? defaultText : service.text,
-    ),
-  );
+  const slotTexts = (slots || []).map(({ defaultText }) => {
+    // Expect defaultText to be in English. When it is provided and we're
+    // displaying English language on the story, set the default text for
+    // this knob to defaultText.
+    // When we switch to a language other than English, set the default
```
Comment (andrew-nowak) [inline packages/utilities/psammead-storybook-helpers/src/input-provider.jsx]:
```suggestion
    // When we switch to a language other than English, set the
```

File: packages/utilities/psammead-storybook-helpers/src/input-provider.jsx [hunk 2]
```diff
@@ -16,18 +16,15 @@ const inputProvider = (slots, componentFunction, services) => () => {
...
+    // text for the knob to the snippet from LANGUAGE_VARIANTS for that
```
Comment (andrew-nowak) [inline packages/utilities/psammead-storybook-helpers/src/input-provider.jsx]:
```suggestion
    // text for the slot to the snippet from LANGUAGE_VARIANTS for that
```

**Review by andrew-nowak — APPROVED** [review]
minor tweak to the comments, but otherwise lgtm

**Review by Bopchy — APPROVED** [review]
LGTM, just needs conflicts to be resolved

**Review by dr3 — APPROVED** [review]

**Pull request merged**